x-pack/filebeat/input/s3/config.go

```diff
@@ -17,6 +17,7 @@ type config struct {
 	QueueURL                  string              `config:"queue_url" validate:"nonzero,required"`
 	VisibilityTimeout         time.Duration       `config:"visibility_timeout"`
 	AwsConfig                 awscommon.ConfigAWS `config:",inline"`
+	ExpendEventListFromField  string              `config:"expand_event_list_from_field"`
```

Comment:
small typo (here and all its uses):

```suggestion
	ExpandEventListFromField  string              `config:"expand_event_list_from_field"`
```

Reply:
😂 thanks! will fix all of them!